### PR TITLE
Automatically redirect to /{api version} if it's missing

### PR DESCRIPTION
**Pull request by magopian**
I got bitten by the documentation saying that I could go to
http://localhost:8000/fields, and had a 404.

@Natim told me that instead of fixing the code, I could automatically redirect
to use the api version if the path is missing it.

If it's a bad idea, then it's @Natim's fault ;)

(Obviously, if it's a good idea, it's thanks to me, right? ;)

### CODE REVIEW

**Comment by Natim** [comment]
Why did you amend the master commit?

**Comment by Natim** [comment]
I think the documentation is another question, but at least this will be useful even if we want to fix the documentation later.

**Comment by Natim** [comment]
I have no specific comments to add to this PR, it looks good to me.

**Comment by leplatrem** [comment]
Yes thanks ! Welcome onboard @magopian !

More generally I'm suprised that pyramid doesn't provide url rewrite (see hard coded urls in the code: https://github.com/spiral-project/daybed/blob/master/daybed/views/models.py#L115-L116)

**Comment by magopian** [comment]
aouch, indeed :/ and thanks ;)

File: daybed/__init__.py [hunk 1]
```diff
@@ -45,6 +49,12 @@ def build_list(variable):
     return [v.strip() for v in variable]
 
 
+def redirect_to_version(request):
+    """Redirect to the current version of the API."""
+    raise httpexceptions.HTTPTemporaryRedirect(
```
Comment (almet) [inline daybed/__init__.py:54]: Thanks for using a 307 here :)

Comment (magopian) [inline daybed/__init__.py:54]: Yeah, it was my idea (of course it wasn't... @natim told me that you would want that, so I did ;)

**Pull request merged**